Resfeber/API/RestManager.swift

```diff
@@ -0,0 +1,297 @@
+//
+//  RestManager.swift
```

Comment:
Consider renaming file to "TeleportManager", or something else more specific so that we can have a separate REST manager for each REST API we use. The backend database with our User info will be a REST API, for example.